soccer/gameplay/plays/offense/kick_gently.py

```diff
@@ -0,0 +1,96 @@
+import play
+import behavior
+import skills.move
+import skills.pivot_kick
+import constants
+import robocup
+import main
+import enum
+import timeout_behavior
+import tactics.coordinated_pass
+import evaluation.touchpass_positioning
+
+
+class KickGently(play.Play):
```

Comment:
Ok, I tested this in simulator and it works pretty much perfectly although there is something strange going on with pivot kick where it will just slowly drive forward with the ball instead of kicking it, I am going to double check to see if this is an issue with pivot kick or with this particular play. Also, it would definitely need some tuning with real robots before we use it in an actual match.